infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/DraftableObjectDao.kt

```diff
@@ -252,13 +276,27 @@ abstract class DraftableDaoBase<T : Draftable<T>>(
             where (o.id = :id or d.id = :id) and o.draft = false
         """
 
+    private fun manyOfficialsFetchSql(table: DbTable) = """
```

Comment:
En yleensä tykkää boolean argumenteista, mutta tässä nuo sql:t menee muutoin toistensa duplikaateiksi niin tuon voisi periaatteessa tehdä sellaisellakin. Eli esim argumenttina `multiFetch: Boolean` ja where-ehdossa `where id ${if (multiFetch) "in (:ids)" else "= :id"}`

Reply:
Juu, hyvältä kuulostava idea